demo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-voice-processor-demo",
-  "version": "2.0.3",
+  "version": "2.0.0",
```

Comment:
We don't publish this anywhere right? So this won't be an issue?

Reply:
correct, to me having such a specific version for the demo just looked weird :)